src/data/actions/assets.js

```diff
@@ -1,16 +1,23 @@
 import { getAssets } from '../../api/client';
 
 export const REQUEST_ASSETS_SUCCESS = 'REQUEST_ASSETS_SUCCESS';
+export const FILTER_UPDATED = 'FILTER_UPDATED';
 
 export const requestAssetsSuccess = response => ({
   type: REQUEST_ASSETS_SUCCESS,
   data: response.assets,
 });
 
-export const requestAssets = courseId =>
+export const requestAssets = assetsFilters =>
   dispatch =>
-    getAssets(courseId, {
-      page: 0,
+    getAssets(assetsFilters.courseId, {
+      page: assetsFilters.page,
+      assetsFilters,
     })
       .then(response => response.json())
       .then(json => dispatch(requestAssetsSuccess(json)));
+
+export const filterUpdate = (filterKey, filterValue) => ({
+  type: FILTER_UPDATED,
+  data: { [filterKey]: filterValue },
```

Comment:
Love the programmatic key assignment syntax, def an underrated ES2015 feature =)